logbook-json/src/main/java/org/zalando/logbook/json/StaticParameterReplacementOperator.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.logbook.json;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.util.function.BinaryOperator;
+
+import static org.zalando.logbook.json.PrimitiveJsonPropertyBodyFilter.quote;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+public class StaticParameterReplacementOperator<T> implements BinaryOperator<String> {
+    private final T replacement;
+
+    private final boolean addQuotation;
```

Comment:
If you split this class into two, you'll see that you no longer need this boolean switch. A fully polymorphic construct (three classes then) would be a lot cleaner.